src/test/java/emissary/util/PlaceComparisonHelperTest.java

```diff
@@ -13,56 +15,65 @@
 import java.util.ArrayList;
 import java.util.List;
 
-class PlaceComparisonHelperTest {
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class PlaceComparisonHelperTest extends UnitTest {
+
+    private static final Class<NullPointerException> NPE = NullPointerException.class;
+
+    private static final byte[] configurationBytes = new StringBuilder()
+            .append("PLACE_NAME = \"TesterPlace\"")
```

Comment:
Don't these need newlines in between the various entries? This is currently generating one long line of muck.

Reply:
> Don't these need newlines in between the various entries? This is currently generating one long line of muck:
> 
> ```
> PLACE_NAME = "TesterPlace"SERVICE_NAME = "TESTER"SERVICE_TYPE = "TRANSFORM"SERVICE_DESCRIPTION = "Place defined for testing"SERVICE_COST = 50SERVICE_QUALITY = 50SERVICE_PROXY = "UNKNOWN"
> ```

ConfigUtil parses the inputStream into separate `ConfigEntry` values just fine as is.